string Class
int64 center_x
int64 center_y
